types/babel-plugin-macros/index.d.ts

```diff
@@ -0,0 +1,36 @@
+// Type definitions for babel-plugin-macros 2.6
+// Project: https://github.com/kentcdodds/babel-plugin-macros
+// Definitions by: Billy Kwok <https://github.com/billykwok>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.6
+import * as Babel from 'babel__core';
```

Comment:
I think this needs to be `@babel/core` (you already have it setup in your path mapping 👍)

Reply:
Thanks for your comment. Fixed.